test/integration/rules/aria-empty-table-header/aria-empty-table-header.html

```diff
@@ -0,0 +1,49 @@
+<html lang="en">
+  <head>
+    <title>empty-table-header test</title>
+    <meta charset="utf8" />
+    <script src="/axe.js"></script>
+  </head>
+
+  <body>
+    <table id="pass1">
```

Comment:
The `id`s should be on the `th`/role elements, otherwise the selectors for the test come out to `table > tr > th`